src/Common/src/Interop/Unix/libcurl/Interop.libcurl_types.cs

```diff
@@ -195,6 +166,20 @@ internal struct curl_version_info_data
             internal int features;
         }
 
+        // Poll values used with curl_multi_wait and curl_waitfd.events/revents
+        internal const int CURL_WAIT_POLLIN = 0x0001;
+        internal const int CURL_WAIT_POLLPRI = 0x0002;
+        internal const int CURL_WAIT_POLLOUT = 0x0004;
+
+#pragma warning disable 0649 // until this file is split up, this produces a warning in the X509 project due to being unused
+        internal struct curl_waitfd
+        {
+            internal int fd;
+            internal short events;
+            internal short revents;
+        };
+#pragma warning restore 0649
+
         public delegate int curl_socket_callback(
```

Comment:
You're right, will delete.  We can bring it back along with some of the others if/when we switch to epoll.